Sources/ArcGISToolkit/Components/Form/FormElementHeader.swift

```diff
@@ -20,7 +20,14 @@ struct FormElementHeader: View {
     let element: FieldFeatureFormElement
     
     var body: some View {
-        Text(element.label)
-            .font(.headline)
+        Group {
+            if element.required {
+                Text(verbatim: "\(element.label) *")
+            } else {
+                Text(element.label)
+            }
+        }
```

Comment:
This seems easier to read and more consider to me.
```suggestion
        Text(verbatim: "\(element.label + (element.required ? " *" : ""))")
```